unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Common/ExploreV2CommonHelpers.cs

```diff
@@ -0,0 +1,18 @@
+/// <summary>
+/// Helpers related to the events management in ExploreV2.
+/// </summary>
+public static class ExploreV2CommonHelpers
```

Comment:
With our new convention this should be a `Util` instead of a `Helper`

Reply:
You're right! Done in https://github.com/decentraland/unity-renderer/pull/1731/commits/cb52e962a43a57293b1e89bcd937916b7dc07824